app/src/main/java/app/grapheneos/camera/ui/activities/MoreSettings.kt

```diff
@@ -169,44 +168,29 @@ class MoreSettings : AppCompatActivity(), TextView.OnEditorActionListener {
             pQField.setText(camConfig.photoQuality.toString())
         }
 
-        pQField.filters = arrayOf(NumInputFilter(this))
-        pQField.setOnEditorActionListener(this)
-
-        iFField = findViewById(R.id.image_format_setting_field)
-        iFField.setOnEditorActionListener(this)
-
-        vFField = findViewById(R.id.video_format_setting_field)
-        vFField.setOnEditorActionListener(this)
-
-        val iPQButton = findViewById<View>(R.id.increase_photo_quality)
-        iPQButton.setOnClickListener {
-
+        pQField.setOnClickListener {
             if (camConfig.photoQuality != NumInputFilter.max) {
-                ++camConfig.photoQuality
                 pQField.setText(camConfig.photoQuality.toString())
             } else {
                 showMessage(
                     "Photo quality can only be between ${NumInputFilter.min} and ${NumInputFilter.max}"
                 )
             }
-        }
 
-        val dPQButton = findViewById<View>(R.id.decrease_photo_quality)
-        dPQButton.setOnClickListener {
             if (camConfig.photoQuality >= NumInputFilter.min) {
-                --camConfig.photoQuality
-
-                if (camConfig.photoQuality >= NumInputFilter.min) {
-                    pQField.setText(camConfig.photoQuality.toString())
-                } else {
-                    pQField.setText("")
-                    showMessage(
-                        "Photo quality was set to auto mode"
-                    )
-                }
+                pQField.setText(camConfig.photoQuality.toString())
             }
         }
 
+        pQField.filters = arrayOf(NumInputFilter(this))
+        pQField.setOnEditorActionListener(this)
+
+        iFField = findViewById(R.id.image_format_setting_field)
+        iFField.setOnEditorActionListener(this)
+
+        vFField = findViewById(R.id.video_format_setting_field)
+        vFField.setOnEditorActionListener(this)
+
```

Comment:
extract to another function like `initInputFields`
probably you may find another functionality which may be added there